services/workflows-service/src/auth/session-serializer.ts

```diff
@@ -8,8 +8,13 @@ export class SessionSerializer extends PassportSerializer {
     super();
   }
 
-  serializeUser(user: User, done: (err: unknown, user: User) => void) {
-    done(null, user);
+  serializeUser(user: User, done: (err: unknown, user: Partial<User>) => void) {
+    done(null, {
+      id: user.id,
+      email: user.email,
+      firstName: user.firstName,
+      lastName: user.lastName,
+    });
```

Comment:
Update the `request.user` type declaration in `declaration.d.ts`
```ts
declare global {
  namespace Express {
    interface User {
      id: string;
      // ...
    }
  }
}
```

Reply:
Done